packages/cactus-plugin-ledger-connector-iroha/src/main/json/openapi.json

```diff
@@ -231,6 +236,82 @@
       "PrometheusExporterMetricsResponse": {
         "type": "string",
         "nullable": false
+      },
+      "IrohaSocketSessionEvent": {
+        "type": "string",
+        "enum": [
+            "org.hyperledger.cactus.api.async.iroha.SocketSession.Subscribe",
```

Comment:
@outSH The same thing here as above regarding the name. Instead of `SocketSession` I'd go with `WatchBlocksV1` for the same reasons.

Reply:
@petermetz Same as above